module-db/Database/Database.cpp

```diff
@@ -95,23 +88,9 @@ Database::Database(const char *name, bool readOnly)
         return;
     }
 
-    /** Workaround for [MOS-351].
-     * Updater app loads databases to wrong directory (1/current/user/db) rather than
-     * (3/user/db). Therefore, we need to check both directories to support both flashing image and updating through
-     * updater.
-     * !!! This workaround should be removed with minor patch of updater, however it cannot be released in one package
-     * with database changes.
-     **/
-    auto filePath = (purefs::dir::getCurrentOSPath() / "user/db");
-    if (!std::filesystem::exists(filePath)) {
-        filePath = (purefs::dir::getUserDiskPath() / "db");
-    }
-
-    LOG_INFO("Running scripts: %s", filePath.c_str());
-    isInitialized_ = initializer->run(filePath.c_str(), InitScriptExtension);
-
-    if (isInitialized_) {
+    if (not isInitialized_) {
```

Comment:
It is a little bit confusing, that we have` isInitialized_` flag `true` here, but before we print _database init done_ info, we do some additional operations.

Reply:
True, fixed.